lib/graphql/schema/union.rb

```diff
@@ -30,6 +31,14 @@ def to_graphql
         def kind
           GraphQL::TypeKinds::UNION
         end
+
+        # Filter possible type based on the current context, no-op by default
+        # @param types [Array<GraphQL::ObjectType>] Types to be filtered
+        # @param ctx [GraphQL::Query::Context] The context for the current query
+        # @param [Array<GraphQL::ObjectType>] the types remaining after the filter is applied
+        def filter_possible_types(types, ctx)
```

Comment:
This default implementation receives but doesn't use `ctx`. Should it be `_ctx` in this case?

Reply:
you are right it should be `_ctx` I originally had these as named args which I couldn't do, but will fix.